### DISCUSSION TITLE
Hoymiles HMT-2250 - 6 Eingaenge, 3 Phasen - Hat jemand Erfahrung damit?

### DISCUSSION THREAD

**Discussion by Manos1966** [hoylabs/OpenDTU-OnBattery#398] [Q&A]
"Fragen kostet nichts", daher frage ich ob jemand bereits Erfahrungen mit dem Hoymiles HMT-2250 und OpenDTUonBattery hat.

PS. Ich weiss, dass ich dafuer ein CMT2300A 868/915MHz brauche (anstatt der NRF24L).

**Comment by genmad** [comment]
@Manos1966 hast du einen HMT in Betrieb? Oder hast du dich aus bestimmten gründen dagegen entschieden?
Hätte auch Interesse an einem HMT.

**Reply by Manos1966** [reply]
Hallo @genmad Noch nicht, ich habe weiterhin meine HM-1500

Wenn ich aber einen neuen kaufen musste, waere einer HMT-2250 (der groesste drei-phasige)
Drei Phasen machen bessere Verteilung der Stroeme als nur eine Phase.
Vor allem wenn man eine grosse Batterie hat (20-30kWh) und dadurch in der Lage ueber mehrere Stunden hohe Stroeme zu liefern.

**Reply by Sorbat** [reply]
Gehst du von der Batterie in alle eingänge des hm-1500? Wieviel kannst du max. von der Batterie einspeisen?

**Reply by Manos1966** [reply]
![HO_HMT-22506T Ampere](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/05268891-b95a-4f7b-b0b1-f787b40fe2a5)

Max. Eingangsstrom pro Eingang 11,5A bei einer 50V Batterie: 11,5x50 = 575W pro Eingang

Rechne realistischer mit 10A pro Eingang, also 10A x 50V = 500W pro Eingang.
Das wurde bedeuten, wenn du alle 6 Eingaenge anschliessen wuerdest, 6 x 500 = 3000W dies ist weit ueber die Leistung deines HMT-2000
Also 4 Eingaenge jeweils 500W = 2000W reichen aus.

Ich weiss nicht ob es besser fuer dein Geraet waere, nur vier Eingaenge anzuschliessen (jeweils Max 10A pro Eingang x50V = 2000W), 
oder alle sechs Eingaenge (jeweils Max 6,7A pro Eingang x50v = 2000W), 
Mein Instinkt wurde sagen, je niedriger die Amperes, desto besser, also alle sechs.

Ebenso wichtig, wie viele A kann deine Batterie aushalten?
Du wirst Max 40A von der Batterie aufnehmen ( 40A x 50V = 2000W)
Was hast du fuer eine Batterie? Pylontech? 
Eine Pylontech 5000 hat zum Beispiel 80A max. Ausgang, daher koennte sie deine 40A Aufnahme locker aushalten.
